definitions/divorce/json/CaseEvent/CaseEvent-prod.json

```diff
@@ -167,13 +67,58 @@
     "SecurityClassification": "Public"
   },
   {
-    "LiveFrom": "01/01/2017",
```

Comment:
@yarpo, I don't quite understand this file. Why do we need a prod file (instead of using the core file)?

Reply:
sometimes we do.

When event exists in prod and non-prod. Can't go to _core_ file as we can't have a duplicate

Reply:
Oh ok. So is that because the prod and non-prod versions have different states?
When do these event get passed to the core file?

Reply:
When it's only in core or only in nonprod.

Core file is used for both prod and nonprod. If we define field/state/event in a core file it must not be defined in prod or nonprod as it doesn't overwrite an existing definition, but it just appends another row.

Rows in excel file cannot be duplicated (in terms of the value of id and service).